Bottom: 3071812b7b338e9abd3a080aefd11452d97fdd34
Top:    5bd0d9c7c2421e4b830432d52c628dcf7099e6d3
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2012-11-10 23:27:33 -0500

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
